src/GitHub.VisualStudio/UI/Options/OptionsControl.xaml

```diff
@@ -0,0 +1,82 @@
+        <Setter.Value>
+          <ControlTemplate TargetType="{x:Type GroupBox}">
+            <Grid SnapsToDevicePixels="true">
+              <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="6" />
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="6" />
+              </Grid.ColumnDefinitions>
+              <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="*" />
+                <RowDefinition Height="6" />
+              </Grid.RowDefinitions>
+              <Border x:Name="Header"
+                      Grid.Row="0"
+                      Grid.RowSpan="2"
+                      Grid.Column="1"
+                      Padding="3,1,3,0">
+                <ContentPresenter ContentSource="Header" RecognizesAccessKey="True" SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" />
+              </Border>
+              <Border Grid.Row="1"
+                      Grid.RowSpan="3"
+                      Grid.ColumnSpan="4"
+                      BorderBrush="Transparent"
+                      BorderThickness="{TemplateBinding BorderThickness}"
+                      CornerRadius="4">
+                <Border.OpacityMask>
+                  <MultiBinding Converter="{StaticResource BorderGapMaskConverter}" ConverterParameter="7">
+                    <Binding ElementName="Header" Path="ActualWidth" />
+                    <Binding Path="ActualWidth" RelativeSource="{RelativeSource Self}" />
+                    <Binding Path="ActualHeight" RelativeSource="{RelativeSource Self}" />
+                  </MultiBinding>
+                </Border.OpacityMask>
+                <Border BorderBrush="{TemplateBinding BorderBrush}" BorderThickness="{TemplateBinding BorderThickness}" CornerRadius="3" />
+              </Border>
+              <ContentPresenter Grid.Row="2"
+                                Grid.Column="1"
+                                Grid.ColumnSpan="2"
+                                Margin="{TemplateBinding Padding}"
+                                SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" />
+            </Grid>
+          </ControlTemplate>
+        </Setter.Value>
+      </Setter>
+    </Style>
+  </UserControl.Resources>
+  <GroupBox Height="85"
+            Margin="8,0,8,0"
+            VerticalAlignment="Top"
+            BorderBrush="LightGray"
+            BorderThickness="1"
+            Padding="6 8 6 8"
+            Style="{DynamicResource GroupBoxFlat}">
+
+    <GroupBox.Header>
+      <Run Text="{x:Static prop:Resources.Options_PrivacyTitle}" />
+    </GroupBox.Header>
+    <DockPanel>
+      <WrapPanel DockPanel.Dock="Top">
+        <CheckBox x:Name="chkMetrics" VerticalAlignment="Center" />
+        <Label Content="{x:Static prop:Resources.Options_MetricsLabel}" />
```

Comment:
We should probably add a hyperlink here that points to https://visualstudio.github.com/samples/. That page shows examples of the type of data we collect.

Here's the [Desktop example](https://desktop.github.com/samples/). Obviously we'll need to add that page to the Visual Studio website, but lets add the link now. :smile: